website/docs/d/netapp_account.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_netapp_account"
+sidebar_current: "docs-azurerm-datasource-netapp-account"
+description: |-
+  Gets information about an existing NetApp Account
+---
+
+# Data Source: azurerm_netapp_account
+
+Use this data source to access information about an existing NetApp Account.
+
+
+## NetApp Account Usage
+
+```hcl
+data "azurerm_netapp_account" "example" {
+  resource_group_name = "acctestRG"
+  name                = "acctestnetappaccount"
+}
+
+output "netapp_account_id" {
+  value = "${data.azurerm_netapp_account.example.id}"
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the NetApp Account.
+
+* `resource_group_name` - (Required) The Name of the Resource Group where the NetApp Account exists.
+
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `location` - Resource location.
```

Comment:
Could we reword this to match the rest of the documentation?

Reply:
Fixed